.github/workflows/clone-db.yaml

```diff
@@ -8,16 +8,16 @@ on:
   workflow_dispatch:
 
 env:
-  DESTINATION_ENVIRONMENT: ms
-  SOURCE_ENVIRONMENT: staging
+  DESTINATION_ENVIRONMENT: staging
+  SOURCE_ENVIRONMENT: stable
   
 
 jobs:
   clone-database:
     runs-on: ubuntu-24.04
     env:
-      CF_USERNAME: ${{ secrets.CF_MS_USERNAME }}
-      CF_PASSWORD: ${{ secrets.CF_MS_PASSWORD }}
+      CF_USERNAME: ${{ secrets.CF_STAGING_USERNAME }}
+      CF_PASSWORD: ${{ secrets.CF_STAGING_PASSWORD }}
     steps:
         - name: Clone Database
```

Comment:
Why are we updating to cf8-cli here?

Reply:
also don't you need to do continue-on-error: true  so that if this fails before unsharing the service the last step is triggered? [reference](https://www.kenmuse.com/blog/how-to-handle-step-and-job-errors-in-github-actions/#:~:text=If%20a%20step%20fails%2C%20the%20job%20and%20workflow%20fails.)